src/transformers/models/ibert/modeling_ibert.py

```diff
@@ -882,17 +883,18 @@ def set_output_embeddings(self, new_embeddings):
     )
     def forward(
         self,
-        input_ids=None,
-        attention_mask=None,
-        token_type_ids=None,
-        position_ids=None,
-        head_mask=None,
-        inputs_embeds=None,
-        labels=None,
-        output_attentions=None,
-        output_hidden_states=None,
-        return_dict=None,
-    ):
+        input_ids: Optional[torch.LongTensor] = None,
+        attention_mask: Optional[torch.FloatTensor] = None,
+        token_type_ids: Optional[torch.LongTensor] = None,
+        position_ids: Optional[torch.LongTensor] = None,
+        head_mask: Optional[torch.FloatTensor] = None,
+        inputs_embeds: Optional[torch.FloatTensor] = None,
+        output_attentions: Optional[bool] = None,
+        output_hidden_states: Optional[bool] = None,
+        return_dict: Optional[bool] = None,
+        labels: Optional[torch.LongTensor] = None,
+        **kwargs: Optional[Dict[str, Any]],
```

Comment:
```suggestion
        **kwargs,
```
In general, I don't think kwargs gets a type annotation in our repo! We might change this later, but for now we can leave it out.

Reply:
fixed in [c813533](https://github.com/huggingface/transformers/pull/16389/commits/c813533a713b0665e7c97ae71dd59ec160fae7b2), but note that in the doc for this model kwargs does get a type annotation

Reply:
That's a fair point - at some point we'll have to figure out a consistent policy on that.